web/src/filter.ts

```diff
@@ -980,7 +980,7 @@ export class Filter {
             // We use join to handle the addition of a comma and space after every name
             // and also to ensure that we return a string and not an array so that we
             // can have the same return type as other cases.
-            return names.join(", ");
+            return util.format_array_as_list(names, "long", "conjunction");
```

Comment:
We can delete delete the comment detailing he previous logic.

Reply:
Done.